test/reviewController.test.js

```diff
@@ -0,0 +1,113 @@
+const { app, connectDB } = require('../src/expressServer'); // Import your Express app here
+const { StatusCodes } = require('http-status-codes');
+const request = require('supertest');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const Review = require('../src/models/Review');
+const User = require('../src/models/User');
+const Album = require('../src/models/Album');
+
+let server;
+let mongooseConnection;
+let mongodb;
+
+beforeAll(async () => {
+  mongodb = await MongoMemoryServer.create();
+  const url = mongodb.getUri();
+  process.env.MONGO_URL = url;
+  mongooseConnection = await connectDB(url);
+  server = await app.listen(8001);
+});
+
+afterAll(async () => {
+  await server.close();
+  await mongooseConnection.disconnect();
+  await mongodb.stop();
+});
+
+describe('ReviewController API Tests', () => {
+  beforeAll(async () => {
+    await Review.deleteMany({});
```

Comment:
Since we're doing this before each test anyway (line 40, `beforeEach` block); it's redundant / not needed to do it at very beginning before the tests run.

Reply:
@akosasante Akos, I was thinking about beforeAll and beforeEach blocks- in most cases propably we can delete   beforeEach block , they are doing practically the same thing - and beforeEach is just a bit more meticulous, isn't it?